Geometry_Engine/Modify/Flip.cs

```diff
@@ -31,6 +31,15 @@ namespace BH.Engine.Geometry
 {
     public static partial class Modify
     {
+        /***************************************************/
+        /**** Public Methods - Vectors                  ****/
+        /***************************************************/
+
+        public static Vector Flip(this Vector vector)
+        {
+            return new Vector { X = -vector.X, Y = -vector.Y, Z = -vector.Z };
+        }
+
```

Comment:
Missing blank line.

Reply:
@pawelbaran are you sure? Line 42 is blank and provides the spacing necessary before line 43 starts the `Public Methods - Curves` statement?

Reply:
Usually we leave 2 blank lines before each big header - or hm, did i miss something?

Reply:
I only ever leave one - I'm always personally against too many blank lines cluttering up a file but if you've been doing 2 previously then that makes sense to keep being consistent :smile:

Reply:
In the end, it will all be refactored by some bot one day :>